dev_guide/application_health.adoc

```diff
@@ -44,6 +44,21 @@ Set a readiness check by configuring the
 
 |===
 
+The exact timing of a probe is controlled by two fields,
+both expressed in units of seconds:
+
+|====
+| Field | Description
+
+| `*initialDelaySeconds*`
+| How long to wait after the container starts to begin the probe.
+
+| `*timeoutSeconds*`
+| How long to wait for the probe to finish (default: `1`).
+If this time is exceeded, OpenShift considers the probe to have failed.
```

Comment:
@nak3 our current version of Kube doesn't include these fields: https://github.com/openshift/origin/blob/08471b135e168e77de716fee9e888fe78cf5f249/Godeps/_workspace/src/k8s.io/kubernetes/pkg/api/types.go#L717

The changes @tnguyen-rh has here look good for now. Also restarting a pod depends on the policy someone has set in it: https://github.com/openshift/origin/blob/08471b135e168e77de716fee9e888fe78cf5f249/Godeps/_workspace/src/k8s.io/kubernetes/pkg/api/types.go#L932